build.gradle.kts

```diff
@@ -294,7 +292,6 @@ tasks {
 							filter {
 								includeTestsMatching("*Demo")
 							}
-							jvmArgs(testJvmArgs)
```

Comment:
Aren't our demo tests running on the module path, why is that obsolete? 😳

Reply:
Are we? That folder doesn't have a `module-info.java` and gradle build has this warning:
```
> Task :demoTests
WARNING: Unknown module: org.junitpioneer specified to --patch-module
WARNING: Unknown module: org.junitpioneer specified to --add-opens
WARNING: Unknown module: org.junitpioneer specified to --add-opens
```

Reply:
Indeed. 👍